File: SAP_APIv1/SAP_API/DTOs/Responses/Order/OrderDetailsResponse.cs
using SAP_API.Models;
using System;
using System.Collections.Generic;


namespace SAP_API.DTOs.Responses.Order
{
    public class OrderDetailsResponse
    {
        public Guid Id { get; set; }
        public DateTime ShouldBeDoneAt { get; set; }
        public OrderStatus Status { get; set; }
        public Customer Customer { get; set; }
        public List<OrderProductResponse> Products { get; set; }
        public List<OrderBakingProgramResponse> BakingPrograms { get; set; }
    }
}

File: bakery-app/src/models/Responses/Order/OrderDetailsResponse.ts
import { OrderStatus } from "../../Enums/OrderStatus";
import Customer from "../../Requests/Customer";
import OrderBakingProgramResponse from "./OrderBakingProgramResponse";
import OrderProductResponse from "./OrderProductResponse";

export default interface OrderDetailsResponse {
    id: string;
    shouldBeDoneAt: Date
    status: OrderStatus;
    customer: Customer;
    products: OrderProductResponse[];
    bakingPrograms: OrderBakingProgramResponse[];
}
